lib/script/script.go

```diff
@@ -50,3 +50,11 @@ func RunCommand(cmd ...string) error {
 	subProcess.Stdout = os.Stdout
 	return subProcess.Run()
 }
+
+// RunCommandSafe executes the given command-line operation, exiting if the command errors
+func RunCommandSafe(cmd ...string) {
```

Comment:
This (super useful) method can probably be used in a lot of other places in the code base.